twilio/Package.md

```diff
@@ -1,51 +1,73 @@
-# Twilio Connector
+Connects to Twilio from Ballerina. 
 
-Allows connecting Twilio REST API.
+# Package Overview
 
-Twilio connector provides a Ballerina API to access the Twilio REST API. This connector provides facility to send SMS, 
-make voice calls, send OTP via SMS and voice call, verify OTP, send user secret QR codes etc. The following section 
-provides you the details on how to use Ballerina Twilio connector.
+This package provides a Ballerina API for the Twilio REST API. It provides the ability to send SMS, make voice calls, 
+send OTP via SMS or call, verify OTP, etc.
 
-## Compatibility
-
-| Ballerina Language Version  | Twilio Basic API Version | Twilio Authy API Version |
-|:---------------------------:|:------------------------:|:------------------------:|
-| 0.970.0-beta12              | 2010-04-01               | v1                       |
-
-## Getting started
+**Basic Operations**
 
-1.  Refer the [Getting Started](https://ballerina.io/learn/getting-started/) guide to download and install Ballerina.
+The `wso2/twilio` package contains operations to get the Twilio account details, send SMS, and make voice calls.
 
-2.  To use Twilio endpoint, you need to provide the following:
+**Authy Operations**
 
-       - Account SId
-       - Auth Token
-       - Authy API Key
+The `wso2/twilio` package contains operations to get Authy app details, add a user, delete a user, get user status, get 
+user secret, request OTP via SMS, request OTP via call, and verify OTP.
 
-       *Please note that, providing Authy API Key is required only if you are going to use Authy related APIs*
-
-3. Create a new Ballerina project by executing the following command.
-
-	```shell
-	   <PROJECT_ROOT_DIRECTORY>$ ballerina init
-	```
+## Compatibility
+|                    |    Version     |  
+| :-----------------:|:--------------:| 
+| Ballerina Language | 0.970.0-beta15 |
+| Twilio Basic API  |    v1         |  
 
-4. Import the Twilio package to your Ballerina program as follows.
+## Sample
+The Twilio connector can be used to send SMS, add a user for Authy, and request OTP via SMS. First, import the 
```

Comment:
Delete "The Twilio connector can be used to send SMS, add a user for Authy, and request OTP via SMS. "

Reply:
Fixed